Fixing a few typos and style issues (#11883)
apache/druid: docs/querying/aggregations.md

> language. For information about aggregators available in SQL, refer to the
> [SQL documentation](sql.md#aggregation-functions).

> Aggregations specified at ingestion time, from within the ingestion spec, provide a way to summarize data before it enters Apache Druid.
> Aggregations are also used at query time.

Reviewer: ```suggestion
You can use aggregations:
-  in the ingestion spec during ingestion to summarize data before it enters Apache Druid.
-  at query time to summarize result data.
```

> The following sections list the types of aggregations available at ingestion and query time.

Reviewer: ```suggestion
The following sections list the available aggregate functions. Unless otherwise noted aggregations are available at both at ingestion and query time.
```
i think they are "aggregate functions" but may be wrong here.

#### `doubleSum` aggregator

> Computes and stores the sum of values as 64-bit floating point value. Similar to `longSum`.

Reviewer: ```suggestion
Computes and stores the sum of values as a 64-bit floating point value. Similar to `longSum`.
```

```json
{ "type" : "doubleSum", "name" : <output_name>, "fieldName" : <metric_name> }
```

#### `floatSum` aggregator

> Computes and stores the sum of values as 32-bit floating point value. Similar to `longSum` and `doubleSum`.

Reviewer: ```suggestion
Computes and stores the sum of values as a 32-bit floating point value. Similar to `longSum` and `doubleSum`.
```

> (Double/Float/Long) First and Last aggregator cannot be used in ingestion spec, and should only be specified as part of queries.

Reviewer: ```suggestion
(Double/Float/Long) Do not use First and Last aggregators in an ingestion spec. They are only supported for queries.
```

| `["dim2"]`               | 2      | (10)                  |
| `[]`                     | 3      | (11)                  |  

> As illustrated in the example, the output number can be thought of as an unsigned _n_ bit number where _n_ is the number of dimensions passed to the aggregator. 
> The bit at position X is set in this number to 0 if a dimension at position X in the aggregator input is included in the sub-grouping. Otherwise, this bit 

Reviewer: ```suggestion
As the example illustrates, you can think of the output number as an unsigned _n_ bit number where _n_ is the number of dimensions passed to the aggregator. 
Druid sets the bit at position X for the number to 0 if the sub-grouping includes a dimension at position X in the aggregator input. Otherwise, Druid set this bit to 1. 
```

Reviewer 2: Should be "Druid sets" in the last sentence!
